.gitignore

```diff
@@ -359,4 +359,6 @@ pipelineassessment.db-shm
 pipelineassessment.db-wal
 
 appsettings.Development.json
-junit.xml
\ No newline at end of file
+junit.xml
+/src/He.PipelineAssessment.UI/appsettings.json
```

Comment:
why are appsettings getting ignored? 🤔

Reply:
Hmm... Not entirely sure.  I'll take a look.